### PR TITLE
update quote API documentation

### PR DESCRIPTION
**Pull request by siong1987**
- smart slippage
- jito tip

### CODE REVIEW

File: openapi/quoteV6.yaml [hunk 1]
```diff
@@ -450,5 +460,23 @@ components:
...
+        type: boolean
+    MaxAutoSlippageParameter:
+      name: maxAutoSlippageBps
+      description: If `autoSlippage` is set to `true`, you can set a max value on the smart slippage the API will suggest using this parameter. It is recommended that you set something here.
+      in: query
+      schema:
+        type: integer
+    AutoSlippageCollisionUsdValueParameter:
+      name: autoSlippageCollisionUsdValue
+      description: If `autoSlippage` is set to `true`, our API will use a default `1000` USD value as way to calculate the slippage impact for the smart slippage. You can set a custom USD value using this parameter.
```
Comment (worlddlckgh) [inline openapi/quoteV6.yaml:479]: do we default on API though?

File: openapi/quoteV6.yaml [hunk 2]
```diff
@@ -450,5 +460,23 @@ components:
...
+    AutoSlippageParameter:
+      name: autoSlippage
+      description: Default is false. By setting this to `true`, our API will suggest a smart slippage that you can use. You should also make sure that you are check out `maxAutoSlippageBps` and `autoSlippageCollisionUsdValue`.
+      in: query
+      schema:
+        type: boolean
+    MaxAutoSlippageParameter:
+      name: maxAutoSlippageBps
+      description: If `autoSlippage` is set to `true`, you can set a max value on the smart slippage the API will suggest using this parameter. It is recommended that you set something here.
+      in: query
```
Comment (worlddlckgh) [inline openapi/quoteV6.yaml:474]:
```suggestion
      description: In conjunction with `autoSlippage=true`, the maximum `slippageBps` returned by the API will respect this value. It is recommended that you set something here.
      in: query
```

File: openapi/quoteV6.yaml [hunk 3]
```diff
@@ -450,5 +460,23 @@ components:
       name: platformFeeBps
       description: If you want to charge the user a fee, you can specify the fee in BPS. Fee % is taken out of the output token.
       in: query
+      schema:
+        type: integer
+    AutoSlippageParameter:
+      name: autoSlippage
+      description: Default is false. By setting this to `true`, our API will suggest a smart slippage that you can use. You should also make sure that you are check out `maxAutoSlippageBps` and `autoSlippageCollisionUsdValue`.
```
Comment (worlddlckgh) [inline openapi/quoteV6.yaml:467]:
```suggestion
      description: Default is false. By setting this to `true`, our API will suggest smart slippage info that you can use. `computedAutoSlippage` is the computed result, and `slippageBps` is what we suggest you to use. Additionally, you should check out `maxAutoSlippageBps` and `autoSlippageCollisionUsdValue`.
```

**Review by worlddlckgh — APPROVED** [review]
Approved, but can you take in my edits/suggestions? thanks

File: openapi/quoteV6.yaml (hunk 1 above)
Comment (siong1987) [reply openapi/quoteV6.yaml:479]: yup. that’s what i see on the code.

**Pull request merged**